activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/converter/EventConverterContext.java

```diff
@@ -29,18 +29,25 @@
 import org.activiti.engine.impl.persistence.entity.ExecutionEntity;
 import org.activiti.engine.runtime.ProcessInstance;
 import org.activiti.engine.task.IdentityLink;
-import org.activiti.engine.task.IdentityLinkType;
 import org.activiti.engine.task.Task;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import static org.activiti.engine.task.IdentityLinkType.CANDIDATE;
+
 @Component
 public class EventConverterContext {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EventConverterContext.class);
 
+    public static final String PROCESS_EVENT_PREFIX = "ProcessInstance:";
+    public static final String TASK_EVENT_PREFIX = "Task:";
+    public static final String TASK_CANDIDATE_USER_EVENT_PREFIX = "TaskCandidateUser:";
+    public static final String TASK_CANDIDATE_GROUP_EVENT_PREFIX = "TaskCandidateGroup:";
+    public static final String ACTIVITI_EVENT_PREFIX = "";
```

Comment:
@constantin-ciobotaru should we call this.. EMPTY_PREFIX? sounds like a hack and it confuses me a lot to have this called ACTIVITI_EVENT_PREFIX

Reply:
Agree, EMPTY_PREFIX is better.